README.md

```
@@ -2,12 +2,17 @@
 
-This application was generated using JHipster 6.5.1, you can find documentation and help at [https://www.jhipster.tech/documentation-archive/v6.5.1](https://www.jhipster.tech/documentation-archive/v6.5.1).
+This application was generated using JHipster 6.5.1, you can find documentation and help
+at [https://www.jhipster.tech/documentation-archive/v6.5.1](https://www.jhipster.tech/documentation-archive/v6.5.1).
 
-This is a "uaa" application intended to be part of a microservice architecture, please refer to the [Doing microservices with JHipster][] page of the documentation for more information.
+This is a "uaa" application intended to be part of a microservice architecture, please refer to
+the [Doing microservices with JHipster][] page of the documentation for more information.
 
-This is also a JHipster User Account and Authentication (UAA) Server, refer to [Using UAA for Microservice Security][] for details on how to secure JHipster microservices with OAuth2.
-This application is configured for Service Discovery and Configuration with Consul. On launch, it will refuse to start if it is not able to connect to Consul at [http://localhost:8500](http://localhost:8500). For more information, read our documentation on [Service Discovery and Configuration with Consul][].
+This is also a JHipster User Account and Authentication (UAA) Server, refer to [Using UAA for Microservice Security][]
```

markdownlint-cli2 (0.22.1)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "using uaa for microservice security"

(MD052, reference-links-images)